html/c-base.css

```diff
@@ -0,0 +1,120 @@
+/********************************************************************************
+macht den c-base c-calendar etwas "spaceiger"
```

Comment:
Dieser Kommentar würde besser als Commit-Message passen. Es wäre schöner, wenn wir das nicht an alle Besucher ausliefern.

Reply:
Ich denke Kommentare machen es anderen Entwicklern etwas leichter den Code anzupassen. Mein Vorschlag wäre
1. alle .css Dateien zusammenfügen (auf Dateipfad achten!)
2. komprimieren mit minify oder dergleichen
3. den daraus resultierenden CSS Code als inline css in der index.html einbinden
4. alle <link... Elemente rauswerfen
5. Kommentar in index.html hinzufügen, wo man die original .css Dateien findet mit Copyright information.
Nachteil, es gibt einen BLOB in der index.html dem man "vertrauen" muss.